src/main/java/org/sonarsource/auth/github/GitHubSettings.java

```diff
@@ -174,6 +178,14 @@ private static String urlWithEndingSlash(@Nullable String url) {
         .type(STRING)
         .defaultValue(valueOf("https://github.com/"))
         .index(index++)
+        .build(),
+      PropertyDefinition.builder(ORGANIZATIONS)
+        .name("Organizations")
+        .description("Only members of these organizations will be able to authenticate to the server. " +
+          "Specify multiple organizations as a comma separated list.")
+        .category(CATEGORY)
```

Comment:
Please use a multi value property (by using `.multiValues(true)`).
You'll then be able to access to the list using `settings.getStringArray()`